src/SearchParameters/index.js

```diff
@@ -182,6 +182,20 @@ SearchParameters.prototype = {
       m.page = 0;
     } );
   },
+
+  /**
+   * typoTolerance setter
+   * Set the value of typoTolerance
+   * @method
+   * @param {boolean} b boolean new value of typoTolerance
+   * @return {SearchParameters}
+   */
+  setTypoTolerance : function setTypoTolerance( b ) {
+    return this.mutateMe( function( m ) {
```

Comment:
It's not a boolean. it's a string with the values "true", "false" , "min" or "strict" http://algolia.github.io/algoliasearch-helper-js/docs/SearchParameters.html#typoTolerance https://www.algolia.com/doc/rest_api#typoTolerance